### DISCUSSION TITLE
Rejecting the requests,ID has already been used.

### DISCUSSION THREAD

**Discussion by SamPandey001** [community/community#63767] [GitHub Education]
### Hi everyone,

I had git edu pack but it expired and when i submit the application it says
```
Unfortunately, we weren't able to approve your educational discount request 
for [@SamPandey001](https://github.com/SamPandey001).
```
My college hasn't provided dif dif I cards in each semester when i use my i card it says 
```
That document has already been used.
```
what should i do??
when i provide dated fees receipt,it doesn't accept.

**Comment by safnimj** [comment]
You can try it by renaming or click a new image of your ID proof
